server/mediaserver.go

```diff
@@ -102,18 +102,16 @@ func NewLivepeerServer(rtmpAddr string, httpAddr string, lpNode *core.LivepeerNo
 
 //StartServer starts the LPMS server
 func (s *LivepeerServer) StartMediaServer(ctx context.Context, maxPricePerSegment *big.Int, transcodingOptions string) error {
-	if s.LivepeerNode.Eth != nil {
-		BroadcastPrice = maxPricePerSegment
-		bProfiles := make([]ffmpeg.VideoProfile, 0)
-		for _, opt := range strings.Split(transcodingOptions, ",") {
-			p, ok := ffmpeg.VideoProfileLookup[strings.TrimSpace(opt)]
-			if ok {
-				bProfiles = append(bProfiles, p)
-			}
+	bProfiles := make([]ffmpeg.VideoProfile, 0)
+	for _, opt := range strings.Split(transcodingOptions, ",") {
+		p, ok := ffmpeg.VideoProfileLookup[strings.TrimSpace(opt)]
+		if ok {
+			bProfiles = append(bProfiles, p)
 		}
-		BroadcastJobVideoProfiles = bProfiles
-		glog.V(common.SHORT).Infof("Transcode Job Price: %v, Transcode Job Type: %v", BroadcastPrice, BroadcastJobVideoProfiles)
 	}
+	BroadcastJobVideoProfiles = bProfiles
+
+	glog.V(common.SHORT).Infof("Transcode Video Profiles: %v", BroadcastJobVideoProfiles)
```

Comment:
The broadcast price was printed before. Shouldn't we still print it now?

Reply:
Broadcast price isn't used right now, but perhaps it will used in the future when we add support for broadcasters to set max prices to use for payments - added it back to the log message!